src/components/carousel/NavigationButton.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC } from 'react';
+import { chakra } from '@chakra-ui/react';
+
+import { ChevronDownLargeIcon } from '../icons';
+import Flex from '../flex';
+
+export type Direction = 'previous' | 'next';
+interface Props {
+  onClick: (direction: Direction) => void;
+  direction: Direction;
+}
+
+const NavigationButton: FC<Props> = ({ direction, onClick }) => {
+  const side = direction === 'previous' ? { left: '0' } : { right: '0' };
+
+  return (
+    <chakra.button
+      onClick={() => onClick(direction)}
+      position="absolute"
+      top="0"
+      {...side}
+      width="lg"
+      height="full"
+      display="flex"
+      justifyContent="center"
+      alignItems="center"
+      color="white"
+      visibility="hidden"
+      _groupHover={{ md: { visibility: 'visible' } }}
+      aria-label={`${direction} slide`}
+    >
+      <Flex
+        justifyContent="center"
+        alignItems="center"
+        borderRadius="sm"
+        width="md"
+        height="md"
+        backgroundColor="gray.900"
+        opacity="40%"
+        _hover={{ opacity: '80%' }}
+      >
+        <ChevronDownLargeIcon
```

Comment:
In Figma the designers created separate icons over rotating them, can you align on that
https://www.figma.com/file/K67jAk1bv1qmnmPVsf2we4/03.-AS24---Icons?node-id=401%3A499

Reply:
so we add separate icons instead of rotating them? This feels a bit wrong to me. What is the reason behind it?

Reply:
I don't know I am fine with rotating as well. But Figma should be consistent with the icons so we don't end up with one left and one top for example. (I have in mind rotating stuff is not the most performant but I would assume that does not matter as long as we don't do very heavy transformations)

Reply:
alright, used separate icons :) wasn't aware of that, thanks